lib/node.js

```diff
@@ -45,7 +45,7 @@ Node.prototype.add = function (item, index, maxWidth) {
     this.values = [];
     this.leaf = false;
     for (var i = 0; i < oldValues.length; i++) {
-      var item = oldValues[i],
+      let item = oldValues[i],
```

Comment:
Non-trivial change here. [See this comment for some storytelling](https://superhuman.slack.com/archives/C05N2TSFVDX/p1695362866382519).